docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -93,10 +93,65 @@ PIDFile=/run/user/1000/overlay-containers/ccfd5c71a088768774ca7bd05888d55cc28769
+272d2813c7988ff26ef3d9386a113e001604150442d1184de74d31d6f02d6bb1
+$ sudo podman create --pod systemd-pod ngix top
+927aadd2d18ece4adbdf8037866d8e146f4f7bbcd7b47029ef27df722195f60a
+$ sudo podman generate systemd --files --name systemd-pod
+/home/user/pod-systemd-pod.service
+/home/user/container-awesome_banach.service
+$ sudo cp pod-systemd-pod.service container-awesome_banach.service /usr/lib/systemd/user
+$ systemctl enable pod-systemd-pod.service
+$ sudo podman pod ps
+POD ID         NAME          STATUS    CREATED          # OF CONTAINERS   INFRA ID
+272d2813c798   systemd-pod   Created   29 minutes ago   2                 6c5d116f4bbe
+$ systemctl start pod-systemd-pod.service
+$ sudo podman pod ps
+POD ID         NAME          STATUS    CREATED          # OF CONTAINERS   INFRA ID
+272d2813c798   systemd-pod   Running   29 minutes ago   2                 6c5d116f4bbe
+$ podman pod ps # 0 Number of pods on user thread.
+$ systemctl stop pod-systemd-pod.service
+$ sudo podman pod ps
+POD ID         NAME          STATUS   CREATED          # OF CONTAINERS   INFRA ID
+272d2813c798   systemd-pod   Exited   29 minutes ago   2                 6c5d116f4bbe
+```
+
+Create systemd unit files for a pod with one simple alpine containers. And Use the generated service file to modify the pod status as a non root uuser. Note: We are not using 'sudo' with our podman commands hence to do any modifications using the .service the systemctl should point to the current user be using --user flag.
+```
+$ podman pod create --name systemd-pod
+0815c7b8e7f5eb6ff637f8f3a61f2072b6cab7591ad663617ee7a1309dfbe382
+$ podman create --pod systemd-pod alpine top
+89967cfd0fbba19a6b8ec9c9c16ed698971338c4ed3d2982540e0f0db8e0d1f9
+$ podman generate systemd --files --name systemd-pod
+/home/user/pod-systemd-pod.service
+/home/user/container-reverent_mahavira.service
+$ sudo cp pod-systemd-pod.service container-reverent_mahavira.service /usr/lib/systemd/user
+$ systemctl --user enable pod-systemd-pod.service
+$ podman pod ps
+POD ID         NAME          STATUS    CREATED          # OF CONTAINERS   INFRA ID
+0815c7b8e7f5   systemd-pod   Created   29 minutes ago   2                 6c5d116f4bbe
+$ systemctl --user start pod-systemd-pod.service
+$ podman pod ps
+POD ID         NAME          STATUS    CREATED          # OF CONTAINERS   INFRA ID
+0815c7b8e7f5   systemd-pod   Running   29 minutes ago   2                 6c5d116f4bbe
+$ sudo podman ps
+CONTAINER ID  IMAGE  COMMAND  CREATED  STATUS  PORTS  NAMES
+$ systemctl --user stop pod-systemd-pod.service
+$ podman pod ps
+POD ID         NAME          STATUS   CREATED          # OF CONTAINERS   INFRA ID
+0815c7b8e7f5   systemd-pod   Exited   29 minutes ago   2                 6c5d116f4bbe
+```
 ## SEE ALSO
-podman(1), podman-container(1), systemctl(1), systemd.unit(5), systemd.service(5)
+podman(1), [podman-container(1)](podman-container.1.md), systemctl(1), systemd.unit(5), systemd.service(5)
```

Comment:
I think that man pages cannot resolve such links.

```suggestion
podman(1), podman-container(1), systemctl(1), systemd.unit(5), systemd.service(5)
```

Reply:
The question is whether this compiles correctly into nroff or not, if it does, this would be kind of nice in markdown since it would become an active link.

Reply:
Valid point! I just checked and it's correctly rendered (i.e., `podman-container(1)`) in the compiled man page.

If we want that, it would be nice to add it to `podman(1)` as well to be consistent.

Reply:
Yes, and then we could move to do this all over the man pages...  But only podman(1) for this PR.

Reply:
@vrothberg thank you for feedback.  I will update the hyperlink ```podman(1)``` working on adding use cases for ```--new``` flag and installation of units. Though quite not sure with enabling lingering for rootless users i read about it and write some appropriate examples

Reply:
Great, thanks a lot, @sujil02! Feel free to ping me on IRC or Slack if you have questions. Or just comment here, as you prefer :)